public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -18,100 +21,113 @@ import { buildBreadcrumbs } from '../../Breadcrumbs/utils';
-          >
-            <Stack gap={0} alignItems="center">
-              <Branding.MenuLogo className={styles.img} />
-              <Icon size="sm" name="angle-down" />
-            </Stack>
-          </ToolbarButton>
-        )}
-        <Breadcrumbs breadcrumbs={breadcrumbs} className={styles.breadcrumbsWrapper} />
-      </Stack>
+    <>
+      <div className={styles.layout}>
+        <Stack minWidth={0} gap={0.5} alignItems="center" flex={{ xs: 2, lg: 1 }}>
+          {!menuDockedAndOpen && (
+            <ToolbarButton
+              narrow
+              id={MEGA_MENU_TOGGLE_ID}
+              onClick={onToggleMegaMenu}
+              tooltip={t('navigation.megamenu.open', 'Open menu')}
+            >
+              <Stack gap={0} alignItems="center">
+                <Branding.MenuLogo className={styles.img} />
+                <Icon size="sm" name="angle-down" />
+              </Stack>
+            </ToolbarButton>
+          )}
+          <Breadcrumbs breadcrumbs={breadcrumbs} className={styles.breadcrumbsWrapper} />
+          {!showToolbarLevel && breadcrumbActions}
+        </Stack>
 
-      <Stack gap={0.5} alignItems="center">
-        <TopSearchBarCommandPaletteTrigger />
-        {unifiedHistoryEnabled && <HistoryContainer />}
-        <QuickAdd />
-        {enrichedHelpNode && (
-          <Dropdown overlay={() => <TopNavBarMenu node={enrichedHelpNode} />} placement="bottom-end">
-            <ToolbarButton iconOnly icon="question-circle" aria-label={t('navigation.help.aria-label', 'Help')} />
-          </Dropdown>
-        )}
-        <ToolbarButton
-          icon="monitor"
-          className={styles.kioskToggle}
-          onClick={onToggleKioskMode}
-          tooltip={t('navigation.kiosk.tooltip', 'Enable kiosk mode')}
-        />
-        {!contextSrv.user.isSignedIn && <SignInLink />}
-        {config.featureToggles.inviteUserExperimental && <InviteUserButton />}
-        {config.featureToggles.extensionSidebar && <ExtensionToolbarItem />}
-        {profileNode && <ProfileButton profileNode={profileNode} />}
-      </Stack>
-    </div>
+        <Stack
+          gap={0.5}
+          alignItems="center"
+          justifyContent={'flex-end'}
+          flex={1}
+          data-testid={!showToolbarLevel ? Components.NavToolbar.container : undefined}
+          minWidth={{ xs: 'unset', lg: 0 }}
+        >
+          <TopSearchBarCommandPaletteTrigger />
+          {unifiedHistoryEnabled && !isSmallScreen && <HistoryContainer />}
+          {!isSmallScreen && <QuickAdd />}
+          {enrichedHelpNode && (
+            <Dropdown overlay={() => <TopNavBarMenu node={enrichedHelpNode} />} placement="bottom-end">
+              <ToolbarButton iconOnly icon="question-circle" aria-label={t('navigation.help.aria-label', 'Help')} />
+            </Dropdown>
+          )}
+          <NavToolbarSeparator />
+          {config.featureToggles.inviteUserExperimental && !isSmallScreen && !actions && <InviteUserButton />}
```

Comment:
Hmm. I have it configured on main, but I couldn't get the InviteUserButton to appear when the `unifiedNavbars` toggle is disabled.

Reply:
@joshhunt you need external user management as well

Reply:
@joshhunt this responsive fix is not related to unified navbars but just a responsive fix for small screens 

On ops right now 
![Screenshot 2025-04-08 at 09 50 34](https://github.com/user-attachments/assets/867e2446-146d-4877-a4fc-810d4401db29)
![Screenshot 2025-04-08 at 09 50 41](https://github.com/user-attachments/assets/f73e7fb2-b3e0-4a3c-b0ce-8225cd715aab)